lib/pacemaker/pcmk_sched_ordering.c

```diff
@@ -863,13 +863,14 @@ order_rsc_sets(const char *id, xmlNode *set1, xmlNode *set2,
     const char *action_1 = crm_element_value(set1, "action");
     const char *action_2 = crm_element_value(set2, "action");
 
-    const char *sequential_1 = crm_element_value(set1, "sequential");
-    const char *sequential_2 = crm_element_value(set2, "sequential");
+    enum pe_ordering flags = pe_order_none;
 
-    const char *require_all_s = crm_element_value(set1, "require-all");
-    bool require_all = require_all_s? crm_is_true(require_all_s) : true;
+    bool require_all;
```

Comment:
this is a case where we really do need to initialize it to false, otherwise the invalid format case leaves it unset

Reply:
Fixed.